collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdapter.java

```diff
@@ -274,18 +293,27 @@ public void clearValues() {
       so in case the player starts a video due to autoplay=true we need to transition into startup state manually
      */
     private void checkAutoplayStartup() {
-        PlaybackConfiguration playbackConfiguration = bitmovinPlayer.getConfig().getPlaybackConfiguration();
-        SourceConfiguration source = bitmovinPlayer.getConfig().getSourceConfiguration();
-        if (playbackConfiguration != null && source != null && source.getFirstSourceItem() != null && playbackConfiguration.isAutoplayEnabled()) {
-            startup();
+        try {
```

Comment:
can be removed, would maybe include a `bitmovinPlayer.getConfig() != null` check though (shouldn't be nullable, but in case that changes)